lib/client-assets.php

```diff
@@ -566,7 +566,13 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	);
 
 	// Initialize the editor.
-	wp_add_inline_script( 'wp-editor', 'wp.api.init().done( function() { wp.editor.createEditorInstance( \'editor\', window._wpGutenbergPost ); } );' );
+	$editor_settings = array(
+		'wideImages' => get_theme_support( 'wide-images' ),
```

Comment:
- Instead of one-off theme supports, we might consider a single umbrella theme support key, whose values is an array with one or more of `wide-images`, etc? Depends if we think "wide-images" has value as a theme support outside just the editor.
- What about themes which support full images, but not wide, or vice-versa? Should we be more granular here?